.rubocop.yml

```diff
@@ -13,3 +13,7 @@ AllCops:
 
 Rails/HasManyOrHasOneDependent:
   Enabled: false
+
+Style/ClassVars:
+  Exclude:
+    - 'spec/support/test_setup.rb'
```

Comment:
Is this your preferred way to exclude a specific test for just one file? I'm more used to including it in the file so it's more obvious.

Reply:
No problem - I will inline the exclusion